tools/generate_latest_map

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env bash
```

Comment:
This comment text is wrong, but in general, since we are only generating this for two types, I'd probably just include them as static files, and not do a `go generate` step.